pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -160,7 +171,8 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 	return f, nil
 }
 
-func updatePluginList(pluginList reflect.Value, pluginSet *config.PluginSet, pluginType reflect.Type, pluginsMap map[string]Plugin) error {
+func updatePluginList(pluginList reflect.Value, pluginSet *config.PluginSet,
```

Comment:
This line break goes against recommended go style https://github.com/golang/go/wiki/CodeReviewComments#line-length

If you don't mind, let me take a look on how to implement this.